x/subscription/client/cli/tx_add_project.go

```diff
@@ -0,0 +1,85 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/lavanet/lava/utils"
+	"github.com/lavanet/lava/x/subscription/types"
+	"github.com/spf13/cast"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdAddProject() *cobra.Command {
```

Comment:
lets add the project with a yml file that contains the keys and policies of the project

Reply:
that way we do it all in one go and we dont need to have another tx